.github/workflows/CI.yml

```diff
@@ -707,13 +709,15 @@ jobs:
 
       - name: Run testing suite on ec2
         if: steps.e2etest-ec2-2.outputs.cache-hit != 'true'
-        run: |
-          if [[ -f testing-framework/terraform/testcases/${{ matrix.testcase }}/parameters.tfvars ]] ; then opts="-var-file=../testcases/${{ matrix.testcase }}/parameters.tfvars" ; else opts="" ; fi
-          cd testing-framework/terraform/ec2 && terraform init && terraform apply -auto-approve -lock=false $opts -var="testing_ami=${{ matrix.testing_ami }}" -var="aoc_version=${{ needs.e2etest-preparation.outputs.version }}" -var="testcase=../testcases/${{ matrix.testcase }}"
-      - name: Destroy resources
-        if: ${{ always() && steps.e2etest-ec2-2.outputs.cache-hit != 'true'}}
-        run: |
-          cd testing-framework/terraform/ec2 && terraform destroy -auto-approve
+        uses: nick-invision/retry@v2
+        with:
+          max_attempts: 3
+          timeout_minutes: 120
```

Comment:
How did we come up with 120 to be an appropriate timeout?

Reply:
Just a shot in the dark. This can be changed.